src/providers/google.ts

```diff
@@ -12,6 +12,7 @@ export function google(
 ): void {
   const DEFAULTS: typeof strategy = {
     scheme: 'oauth2',
+    codeChallengeMethod: '',
```

Comment:
S256 seems to be what Google prefers: https://developers.google.com/identity/protocols/oauth2/native-app

Reply:
But it doesn't work.

Reply:
What do you mean? Why?

Reply:
S256 doesn't work properly in the context, why I don't know. There is a lot on stsckoverflow about this and the only way to solve this is to leave it empty.